README.md

```diff
@@ -108,6 +108,10 @@ project:
 ```clojure
 :cljfmt {}
 ```
+#### Leiningen specific configuration
+* `:load-dotfile?` -
+  true if cljfmt read configuration from `.cljfmt.edn` or `.cljfmt.clj`.
+  Defaults to false.
```

Comment:
Let's expand on this a little:

```markdown
  true if the CLI configuration file (either `.cljfmt.edn` or `.cljfmt.clj`)
  should be merged with the configuration options in the Leiningen project
  file. The Leiningen options always take priority. Defaults to false.
```

Reply:
> The Leiningen options always take priority

I don't think that is true as written.  Should I swap the merge order here 

```
(defn- merge-file-options [{:keys [load-dotfile?] :as options}]
  (main/merge-options options (when load-dotfile? (main/load-configuration))))

```

Reply:
Ah, yes, the Leiningen options should take priority, as that allows you the option to override the dotfile if you want.